<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>疲劳驾驶监测系统</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="openMyCom3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>人脸检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMyCom3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openMyCom5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>监控心率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMyCom5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭心率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openMyCom1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>监控定位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMyCom1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭定位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openMyCom2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>脑电检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMyCom2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭脑电</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>心跳间隔</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openGPRS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>12</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeGPRS">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>12</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出登陆</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>295</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GPRS界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>北斗数据</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>经度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>纬度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>验证</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>400</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>可连蓝牙</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_info">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_heart">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>脑电专注度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_tgam">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>perclos值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>眼部疲劳等级</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_eye">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_eyelevel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_yawnlevel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_yawn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>60s哈欠次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>哈欠疲劳等级</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_heart_change">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HRV</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
